app/views/shared/_sidebar.html.erb

```diff
@@ -36,7 +36,7 @@
         <ul class="menu">
           <%= render "/shared/sidebar_content/dashboard" unless authorized_user.can_advertise_and_publish? %>
           <%= render "/shared/sidebar_content/dashboards" if authorized_user.can_advertise_and_publish? %>
-          <%= render "/shared/sidebar_content/organizations" if authorized_user.organizations.exists? || authorized_user.can_admin_system? %>
+          <%= render "/shared/sidebar_content/teams" if authorized_user.organizations.exists? || authorized_user.can_admin_system? %>
```

Comment:
Looks like `teams` verbiage is leaking into the source impl. ಠ_ಠ

Reply:
I'M SORRY 😳😂